sygnal/pipeline.yml

```diff
@@ -27,6 +27,25 @@ steps:
           image: "python:3.7"
           mount-buildkite-agent: false
 
+
+  - command:
+      - "python -m pip install 'isort~=5.0'"
+      - "isort --check-only --diff ."
```

Comment:
could you set up `tox` to do this instead, so that we don't have to hardcode isort and mypy versions and commandlines into the pipeline repo, and so that it's easier to run the commands yourself manually from a working copy?

See the synapse repo for examples.